### DISCUSSION TITLE
prefix

### DISCUSSION THREAD

**Discussion by primej8** [jagrosh/MusicBot#1593] [Troubleshooting] [answered]
### Bug Description

bot only responds if the prefix is @mention

### Steps to Reproduce

set the prefix to @mention
works fine and plays

set the prefix to !!
commands stop working

### Expected Result

commands to work with !!

### Debug Output

System Properties:
  java.version = 22.0.1
  java.vm.name = Java HotSpot(TM) 64-Bit Server VM
  java.vm.specification.version = 22
  java.runtime.name = Java(TM) SE Runtime Environment
  java.runtime.version = 22.0.1+8-16
  java.specification.version = 22
  os.arch = amd64
  os.name = Windows 10

JMusicBot Information:
  Version = 0.4.1
  Owner = (me)
  Prefix = @mention
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.4.1_353
  JDA-Utilities Version = 3.0.5
  Lavaplayer Version = 2.1.1

Runtime Information:
  Total Memory = 34
  Used Memory = 20

Discord Information:
  ID = 1250056527742500934
  Guilds = 1
  Users = 2

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by MichailiK** [comment, marked as answer]
2 things:
1. in your config.txt, make sure you set `prefix = "!!"`.
2. You need to go to your bot settings on the Discord Developer dashboard, and make sure you have "Message Content Intent" ticked.

![image](https://github.com/jagrosh/MusicBot/assets/39029839/37ba4b73-ed41-4733-9a76-de3f4f42130f)

**Reply by primej8** [reply]
thnks works now

**Reply by primej8** [reply]
is there any way to have it use / command that shows up in the auto-complete?

**Reply by MichailiK** [reply]
Slash commands are being worked on. We'll hopefully implement them in JMusicBot 0.5.0.

**Reply by primej8** [reply]
Ok
